aries_cloudagent/protocols/issue_credential/v2_0/handlers/cred_issue_handler.py

```diff
@@ -68,7 +68,11 @@ async def handle(self, context: RequestContext, responder: BaseResponder):
         # Automatically move to next state if flag is set
         if context.settings.get("debug.auto_store_credential"):
             try:
-                cred_ex_record = await cred_manager.store_credential(cred_ex_record)
+                cred_ex_record = await cred_manager.store_credential(
+                    cred_ex_record,
+                    supplement=context.message.supplements,
+                    attachment=context.message.attachments,
+                )
```

Comment:
I don't think there's anything that should be changed in this PR right now but just to note quickly: for the issuer side, we need to add the supplements and attachments to the credential exchange record (to hold onto the values between steps in the protocol). It likely makes sense for `cred_manager.store_credential` to accept just the `cred_ex_record` again when we have those changes finalized.